lms/views/lti/basic_launch.py

```diff
@@ -239,9 +240,26 @@ def _record_launch(self):
         self.request.find_service(name="application_instance").update_from_lti_params(
             self.context.application_instance, self.request.lti_params
         )
-
         if not self.request.lti_user.is_instructor and not self.context.is_canvas:
             # Create or update a record of LIS result data for a student launch
             self.request.find_service(name="grading_info").upsert_from_request(
                 self.request
             )
+
+    def _check_inclusive_access(self):
```

Comment:
Maybe this should be eventually `check_license` or similar of which VS inclusive access is just one of the possibilities.